index.bs

```diff
@@ -131,7 +131,8 @@ session=] can contain multiple [=WebTransport streams=].
 
 ## WebTransport session ## {#webtransport-session}
 
-A <dfn for="protocol">WebTransport session</dfn> is a session of WebTransport over HTTP/3.
+A <dfn for="protocol">WebTransport session</dfn> is a session of WebTransport over an HTTP/3
+or HTTP/2 <dfn>underlying [=connection=]</dfn>.
```

Comment:
Now that we have this [=underlying connection=] concept, should we also use it at https://pr-preview.s3.amazonaws.com/jan-ivar/webtransport/pull/547.html#ref-for-concept-connection%E2%91%A1?